state_experiments/plot_state_results.py

```diff
@@ -0,0 +1,127 @@
+import numpy as np
+import matplotlib.pyplot as plt
+import pickle
+import pandas as pd
+import seaborn as sns
+from gerrychain import Graph, GeographicPartition, Partition, Election, accept
+from gerrychain.updaters import Tally, cut_edges
+import glob
+import functools
+import operator
+import argparse
+
+parser = argparse.ArgumentParser(description="Plot State Run Results", 
+                                 prog="plot_state_results.py")
+parser.add_argument("state", metavar="state id", type=str,
+                    choices=["VA", "TX", "CO", "LA", "NM"],
+                    help="which state to run chains on")
+parser.add_argument("race_col", metavar="Race_column", type=str,
+                    choices=["BVAP", "HVAP", "nWVAP"],
+                    help="Which race_col")
+args = parser.parse_args()
+
+
+num_h_districts = {"VA": 100, "TX": 150, "AR": 100, "CO": 65, "LA": 105, "NM": 70}
+pop_bal = {"CO": 2.0, "LA": 4.5, "NM": 4.5, "TX": 2.0, "VA": 2.0}
```

Comment:
Same, could you leave a comment about what `pop_bal` is? I don't think it is obvious that these are % deviations to readers who are not us.